signup-service/src/main/java/io/mosip/signup/services/RegistrationService.java

```diff
@@ -324,7 +324,7 @@ public RegistrationStatusResponse updatePassword(ResetPasswordRequest resetPassw
         transaction.setRegistrationStatus(RegistrationStatus.PENDING);
         cacheUtilService.setStatusCheckTransaction(transactionId, transaction);
 
-        notificationHelper.sendSMSNotificationAsync(resetPasswordRequest.getIdentifier(), transaction.getLocale(),
+        notificationHelper.sendSMSNotificationAsync(resetPasswordRequest.getIdentifier(), resetPasswordRequest.getLocale(),
```

Comment:
if the language code is null in the request, we can take the language code from the transaction. what do you think @mengleang-0090?